tracing/go.mod

```diff
@@ -0,0 +1,26 @@
+module goa.design/examples/tracing
+
+go 1.21.3
+
+// Temporarily replacing as the official release is not yet available
+replace goa.design/examples/basic => github.com/ikawaha/calc v0.0.3
```

Comment:
Should this be removed?

Reply:
Yes, this `replace` directive is temporary until the official release of `goa.design/examples/basic` becomes available. Once the official release is out, we can remove this replacement.

Reply:
Would it be okay to remove it in this PR, even if it might cause the tests to fail?

Reply:
To address your feedback regarding the replace directive, I've made adjustments to ensure a smoother build process. Specifically, I've removed the `tracing` sample from this PR (https://github.com/goadesign/examples/pull/130/commits/772a41ac641d5bad0fc12a3b37a49078b0123c8d). Instead, I've prepared a separate PR specifically for adding the tracing sample(https://github.com/ikawaha/examples/pull/1). This eliminates the need to push the noted replace directive. If it works for you, could we merge this PR first and then move on to merging the subsequent PR for `tracing`?

Reply:
Perfect, this looks great.